cluster/gce/gci/configure-kubeapiserver.sh

```diff
@@ -15,24 +15,28 @@
 
 
 # Configures etcd related flags of kube-apiserver.
+# ETCD_SERVERS and ETCD_SERVERS_OVERRIDES are used in tests.
 function configure-etcd-params {
   local -n params_ref=$1
+  local -r host_ip=$(hostname -i)
```

Comment:
Should this be controlled using an env param? If the env param is missing then we fallback to the old behavior? I am worried about not having a fallback plan in case this fails for some reason.

Reply:
Added fallback logic.